include/couch_mrview.hrl

```diff
@@ -98,7 +98,8 @@
     should_close = false,
     buffer = [],
     bufsize = 0,
-    threshold = 1490
+    threshold = 1490,
+    sent = false
```

Comment:
clearly it's confusing, because `metadata_sent` would be a worse name. `row_sent` would be more accurate (we're trying to suppress sending meta if we've already sent a row).